snowdialect.py

```diff
@@ -583,7 +583,21 @@ def get_table_comment(self, connection, table_name, schema=None, **kw):
                                     )
         cursor = connection.execute(sql_command)
         ans = cursor.fetchone()
-        return {'text': ans['comment'] if ans['comment'] else None}
+        if ans is None:
+            # NOTE: this may happen because the "table" being reflected is
+            # actually a view; SQLAlchemy has the same code path for both.
+            sql_command = "SHOW /* sqlalchemy:get_view_comment */ " \
+                "VIEWS LIKE '{0}'{1}".format(
+                    table_name,
+                    (' IN SCHEMA {0}'.format(self.normalize_name(schema))) if schema else ''
+                )
+            cursor = connection.execute(sql_command)
+            ans = cursor.fetchone()
```

Comment:
Could you make this a separate function, like `get_view_comment`?

Reply:
Please install `tox` and make sure `tox -e fix_lint` passes.

Reply:
Actually, don't worry about the `fix_lint`, it's made to work on our internal repo. I'll deal with linting issues.